src/main/java/nl/tudelft/jpacman/npc/ghost/DispersionBlinky.java

```diff
@@ -0,0 +1,18 @@
+package nl.tudelft.jpacman.npc.ghost;
+
+import nl.tudelft.jpacman.board.Direction;
+
+public class DispersionBlinky extends DispersionMode{
+
+	Ghost g;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/mikis-millemaci/jpacman-framework/file/2532399316/issues/source?bid=3256381&fileBranchId=3284358#l7)